modules/ossm-config-web-console.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/customizing-installation-ossm.adoc
+// * service_mesh/v2x/customizing-installation-ossm.adoc
+
+[id="ossm-config-web-console_{context}"]
+= Configuring the control plane with the web console
+
+Follow this procedure to configure the `ServiceMeshControlPlane` by using the {product-title} web console.
```

Comment:
```suggestion
You can configure the `ServiceMeshControlPlane` by using the {product-title} web console.
```

Reply:
Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes".
Instead, focus on the purpose or value of the content, and include words that users are likely to search
for. For example, in a short description in an online help topic, provide users with enough information to
determine whether the topic meets their needs. You can use self-referential text in learning modules
and tutorials.

Reply:
Multiple instances of "Follow this procedure" and "this section."

Reply:
@rolfedh Removed a bunch of instances of "follow this procedure." I left a couple of instances and and a this section that will be changed in a subsequent PR.